### PR TITLE
feat: hover on username

### PR DESCRIPTION
**Pull request by sahas-01**
Hovering on username updated as per the designs.

### CODE REVIEW

**Review by alphainfinitus — APPROVED** [review]

File: src/ui-components/IdentityBadge.tsx [hunk 1]
```diff
@@ -50,18 +55,71 @@ const IdentityBadge = ({ className, address, identity, flags, web3Name }: Props)
...
+			<span className='text-xs text-bodyBlue font-normal pt-0.5'>{identity.email}</span>
+		</li>
+		}
+		{(identity?.judgements?.length || 0) > 0 &&
+		<li className='flex items-center'>
+			<span className='desc flex items-center text-sm text-bodyBlue font-medium'><JudgementIcon className='mr-1.5'/>Judgements:</span>
+			<span className='text-xs text-bodyBlue'>{displayJudgements}</span>
+		</li>
+		}
+		{identity?.pgp &&
```
Comment (alphainfinitus) [inline src/ui-components/IdentityBadge.tsx:82]: no pgp icon

Comment (alphainfinitus) [reply src/ui-components/IdentityBadge.tsx:82]: spy icon

File: src/ui-components/IdentityBadge.tsx [hunk 2]
```diff
@@ -50,18 +55,71 @@ const IdentityBadge = ({ className, address, identity, flags, web3Name }: Props)
...
+		}
+		{identity?.web &&
+		<li className='flex items-center'>
+			<span className='desc text-bodyBlue font-medium flex text-sm'><WebIcon className='mr-1.5 mt-1 -ml-0.5' />Web: </span>
+			<span className='text-xs text-bodyBlue font-normal pt-0.5'>{identity.web}</span>
+		</li>
+		}
+		{flags?.isCouncil &&
+		<li className='flex items-center'>
+			<span className='desc text-bodyBlue font-medium text-sm'>Council member: </span><CouncilEmoji/>
```
Comment (alphainfinitus) [inline src/ui-components/IdentityBadge.tsx:108]: Council emoji should be first

**Review by alphainfinitus — CHANGES_REQUESTED** [review]
Please check comments

File: src/ui-components/IdentityBadge.tsx (hunk 2 above)
Comment (sahas-01) [reply src/ui-components/IdentityBadge.tsx:108]: Resolved, pls check

File: src/ui-components/IdentityBadge.tsx (hunk 1 above)
Comment (sahas-01) [reply src/ui-components/IdentityBadge.tsx:83]: Added, pls check

**Review by alphainfinitus — CHANGES_REQUESTED** [review]
Please check comments

File: src/ui-components/IdentityBadge.tsx [hunk 3]
```diff
@@ -50,18 +56,71 @@ const IdentityBadge = ({ className, address, identity, flags, web3Name }: Props)
...
+		}
+		{identity?.web &&
+		<li className='flex items-center'>
+			<span className='desc text-bodyBlue font-medium flex text-sm'><WebIcon className='mr-1.5 mt-1 -ml-0.5' />Web: </span>
+			<span className='text-xs text-bodyBlue font-normal pt-0.5'>{identity.web}</span>
+		</li>
+		}
+		{flags?.isCouncil &&
+		<li className='flex items-center'>
+			<span className='desc text-bodyBlue font-medium text-sm'><CouncilEmoji /> Council member: </span>
```
[the remark this answers is not in the document]
Comment (sahas-01) [reply src/ui-components/IdentityBadge.tsx:109]: Done, pls check

**Review by alphainfinitus — APPROVED** [review]

**Pull request merged**